docs/business.yaml

```diff
@@ -6304,6 +6304,11 @@ components:
         isFutureEffective:
           type: boolean
           description: Identify if filing was submitted with a future effective date (yes/no).
+        paymentDate:
+          type: string
+          minLength: 1
```

Comment:
paymentDate can be null. @argush3 do we need to remove this?

Reply:
maybe add a note indicating that filings from legacy filings won't have a value